.changelog/4557.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+core: Migrate to CRT.
```

Comment:
Have other products included this in their CHANGELOG? It doesn't seem user-facing so personally I would not include it, as our end-users have no real concept of what CRT is and how it applies to them (it doesn't)

Reply:
They've not, the Changelog-checker workflow was failing so I added this but I can definitely remove this